### ISSUE TITLE
GNOME audio controls use wrong device after being device is plugged in 

### ISSUE THREAD

**Issue by RyanSquared** [pop-os/pop#605]
**Distribution (run `cat /etc/os-release`):**
 
 ```
NAME="Pop!_OS"
VERSION="19.04"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 19.04"
VERSION_ID="19.04"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=disco
UBUNTU_CODENAME=disco
```
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

```
gnome-shell:
  Installed: 3.32.1-1ubuntu1~19.04.1pop1~1560527848~19.04~6c4cbae
  Candidate: 3.32.1-1ubuntu1~19.04.1pop1~1560527848~19.04~6c4cbae
  Version table:
 *** 3.32.1-1ubuntu1~19.04.1pop1~1560527848~19.04~6c4cbae 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu disco/main amd64 Packages
        100 /var/lib/dpkg/status
     3.32.1-1ubuntu1~19.04.1 500
        500 http://us.archive.ubuntu.com/ubuntu disco-updates/main amd64 Packages
     3.32.0+git20190410-1ubuntu1 500
        500 http://us.archive.ubuntu.com/ubuntu disco/main amd64 Packages
```

**Issue/Bug Description:**

GNOME changes the "active" audio device to be one that is plugged in, but the plugged-in device is muted.

**Steps to reproduce (if you know):**

1. Take note of the device last used for output, this can be "Internal Speakers" or "Headphones"
2. Plug in a USB cable with an audio output. The one I'm using has four - my microphone (which has an aux output) and my DAC both have an analog and digital output.
3. Go to the GNOME audio settings and note that the "output device" is not the same as it was before.
4. Load up a video on YouTube or similar site and observe that audio is playing through the device that was previously selected, not the one that is currently selected.
5. Unmute and mute using media keys and GNOME will mute the old audio output and then unmute the new audio output.

**Expected behavior:**

GNOME either unmutes a single audio output (preferably with the option to disable such functionality) or continues assuming the still-unmuted device is the one the user wants to control.

**Other Notes:**

This issue may be specific to the fact I have (at least) two audio outputs connecting to the same time, but I can't test a single audio output as I don't have anything that only supports one.

**Comment by isantop** [comment]
I'm having issues replicating this issue on my machine here, testing on an Oryx 4 and using the internal speakers, the headphone jack, and an external USB DAC. I did notice that the system tries to auto-select newly plugged in USB Audio devices, and that the playback to that device lags by a few seconds (5-10 ish from my count). Perhaps this behavior is related to the problem?

**Comment by RyanSquared** [comment]
> and that the playback to that device lags by a few seconds (5-10 ish from my count)

Surprisingly, throughout all of my testing, it seems this lined up *exactly* with when I was pressing buttons. In addition, it looked like another program I was running seemed to not take changes into effect at all (even if I went and changed it myself, and it worked in Chromium, it wouldn't work if I went back to the other program). I'll close this for now, will re-open if I see any other oddities.
